db/migrate/20191108135108_add_service_credential_tables.rb

```diff
@@ -0,0 +1,58 @@
+class AddServiceCredentialTables < ActiveRecord::Migration[5.2]
+  def change
+    create_table :service_credential_types, :id => :bigserial, :force => :cascade do |t|
```

Comment:
What are the different credential types?  Is this something that needs to be a separate model or can it just be a string column on the credential table

Reply:
The important thing is the "extra" column, which contains form input's description of credentials - needed for ordering the service with "prompt on launch" on credentials

Reply:
@agrare these input types are not used for ordering, so I removed credential types at all as you wrote